### DISCUSSION TITLE
about nerd font culling

### DISCUSSION THREAD

**Discussion by Ceobe33** [LazyVim/LazyVim#599] [Q&A]
![image](https://user-images.githubusercontent.com/72852768/232352230-fb462bb4-a384-4d5e-9486-eb2ccac6721e.png)  
these character `m`, `d` of `vim.cmd` seems be culled, i have try to replace font type, but didn't work ,  
and `m` in `keymap` is normal
how can i solve it?  
thanksss

**Comment by robsonpeixoto** [comment]
Watch this amazing video, maybe can help you: https://www.youtube.com/watch?v=mQdB_kHyZn8
